### PR TITLE
Ignore 390111

### PR DESCRIPTION
**Pull request by smtakeda**
Ignore 390111 when closing session.

### CODE REVIEW

File: Snowflake.Data.Tests/SFBaseTest.cs [hunk 1]
```diff
@@ -19,8 +19,8 @@ namespace Snowflake.Data.Tests
     [SetUpFixture]
     public class SFBaseTest
     {
-        private const string connectionStringFmt = "scheme=https;host={0};port=443;" +
-            "user={1};password={2};account={3};role={4};db={5};schema={6};warehouse={7}";
+        private const string connectionStringFmt = "scheme={0};host={1};port={2};" +
+            "user={3};password={4};account={5};role={6};db={7};schema={8};warehouse={9}";
```
Comment (smtakeda) [inline Snowflake.Data.Tests/SFBaseTest.cs]: Fixed to support local test.

File: Snowflake.Data.Tests/SFBaseTest.cs [hunk 2]
```diff
@@ -101,6 +106,15 @@ public class TestConfig
...
         [JsonProperty(PropertyName = "SNOWFLAKE_TEST_ROLE", NullValueHandling = NullValueHandling.Ignore)]
         internal string role { get; set; }
+
+        [JsonProperty(PropertyName = "SNOWFLAKE_TEST_PROTOCOL", NullValueHandling = NullValueHandling.Ignore)]
+        internal string protocol { get; set; }
+
+        public TestConfig()
+        {
+            this.protocol = "https";
+            this.port = "443";
```
Comment (smtakeda) [inline Snowflake.Data.Tests/SFBaseTest.cs]: default values.

File: Snowflake.Data.Tests/SFConnectionIT.cs [hunk 3]
```diff
@@ -29,8 +29,11 @@ public void TestBasicConnection()
                 Assert.AreEqual("", ((SnowflakeDbConnection)conn).DataSource);
 
                 string serverVersion = ((SnowflakeDbConnection)conn).ServerVersion;
-                string[] versionElements = serverVersion.Split('.');
-                Assert.AreEqual(3, versionElements.Length);
+                if (!string.Equals(serverVersion, "Dev"))
```
Comment (smtakeda) [inline Snowflake.Data.Tests/SFConnectionIT.cs]: The local GS returns "Dev".

File: Snowflake.Data/Core/SFSession.cs [hunk 4]
```diff
@@ -207,10 +207,10 @@ internal void close()
                 authorizationToken = string.Format(SF_AUTHORIZATION_SNOWFLAKE_FMT, sessionToken)
             };
           
-            var response = restRequester.Post<NullDataResponse>(closeSessionRequest);
+            var response = restRequester.Post<CloseResponse>(closeSessionRequest);
             if (!response.success)
             {
-                logger.Warn($"Failed to delete session, error ignored. Code: {response.code} Message: {response.message}");
+                logger.Debug($"Failed to delete session, error ignored. Code: {response.code} Message: {response.message}");
```
Comment (smtakeda) [inline Snowflake.Data/Core/SFSession.cs]: All errors are ignored.

**Comment by ChTimTsubasa** [comment]
@smtakeda Remind: Be careful the destination branch. You would like to merge this PR into prep-1.0.10 rather than master.

**Comment by smtakeda** [comment]
Good point. Let me recreate PR.

**Pull request closed without merging**